src/plugins/dam/components/search-assets-screen/index.js

```diff
@@ -96,11 +99,12 @@ function SearchAssetsScreen(props) {
                 const canUpdate = policy.isGranted(`${schema.getQName()}:update`);
                 const seq = node.get('gallery_seq');
                 const transcodingStatus = node.get('transcoding_status');
+                const handleNodeClick = nodeClickNavigator(navigate, node);
 
                 return (
-                  <tr key={`${node.get('_id')}`} className={`status-${node.get('status')}`}>
-                    <td><Input type="checkbox" onChange={() => batch.toggle(node)} checked={batch.has(node)} /></td>
-                    <td className="text-center"><AssetIcon id={node.get('_id')} /></td>
+                  <tr key={`${node.get('_id')}`} className={`status-${node.get('status')}`} style={{cursor: 'pointer'}} onClick={handleNodeClick}>
```

Comment:
@rsumilang you won't need this style, i tested a fix on sloth branch and it seems to work in chrome.
```
<tr key={`${node.get('_id')}`} className={`cursor-pointer status-${node.get('status')}`}>
```